app/views/campaign_pages/index.slim

```diff
@@ -11,7 +10,7 @@
           th Actions
           th
       tbody
-        - @campaign_pages.each do |page|
+        - [*@campaign_pages].each do |page|
```

Comment:
what does this do? why iterate over a list of a splat instead of the list itself?